app/javascript/elm/src/SessionsList.elm

```diff
@@ -0,0 +1,192 @@
+                selectedSessionId =
+                    List.head << List.map .id << List.filter .selected
+            in
+            ( { model | sessions = sessions, selectedSessionId = selectedSessionId sessions }, Cmd.none )
+
+        LoadMoreSessions ->
+            ( model, loadMoreSessions () )
+
+
+
+---- VIEW ----
+
+
+viewSession : Maybe Int -> Session -> Html Msg
+viewSession selectedSessionId session =
+    li
+        [ style "padding" "5px 10px"
+        , style "clear" "both"
+        , style "overflow" "hidden"
+        ]
+        [ input
+            [ type_ "radio"
+            , id <| "radio-" ++ fromInt session.id
+            , checked <| selectedSessionId == Just session.id
+            , onClick <| ToggleSessionSelection session.id
+            , style "float" "left"
+            , style "margin" "3px 0 0"
+            ]
+            []
+        , dl
+            [ style "float" "left"
+            , style "margin" "0 0 0 10px"
+            ]
+            [ dt
+                [ style "font-size" "14px"
+                , style "color" "#000"
+                , style "font-weight" "normal"
+                ]
+                [ label
+                    [ class "narrow"
+                    , style "display" "block"
+                    , style "cursor" "pointer"
+                    , style "width" "170px"
+                    , for <| "radio-" ++ fromInt session.id
+                    ]
+                    [ text session.title ]
+                ]
+            , dd
+                [ style "font-size" "11px"
+                , style "margin" "0"
+                , style "color" "#000"
+                , style "font-weight" "normal"
+                ]
+                [ label
+                    [ class "narrow"
+                    , style "display" "block"
+                    , style "cursor" "pointer"
+                    , style "width" "170px"
+                    ]
+                    ([ div [] [ text <| session.username ++ ", " ++ session.timeframe ]
+                     ]
+                        ++ List.indexedMap (viewShortType <| List.length session.shortTypes) session.shortTypes
+                    )
+                ]
+            ]
+        ]
+
+
+viewShortType : Int -> Int -> ShortType -> Html msg
+viewShortType length index shortType =
+    span [ class shortType.type_ ]
+        [ text shortType.name
+        , span []
+            [ if index == length - 1 then
+                text ""
+
+              else
+                text "/"
+            ]
+        ]
+
+
+view : Model -> Html Msg
+view model =
+    ul
+        []
+        (List.map (viewSession model.selectedSessionId) model.sessions
+            ++ [ viewLoadMore <| List.length model.sessions ]
+        )
+
+
+viewLoadMore : Int -> Html Msg
+viewLoadMore sessionCount =
+    if sessionCount /= 0 && modBy 50 sessionCount == 0 then
+        li [] [ button [ onClick LoadMoreSessions ] [ text "Load More..." ] ]
+
+    else
+        text ""
```

Comment:
So if we have exactly 50 sessions available we show the load more button, even though there is nothing else to load? Maybe that's fine, I know this loading more sessions is tricky and it might be impossible to get do it correctly right now. Just wanted to notice that, so we remember to fix it if this is simplified in the future.

Reply:
Yeah, this sucks. I just brought over what we have in js for now